src/sentry/rules/conditions/event_attribute.py

```diff
@@ -256,7 +256,7 @@ def _handle(cls, path: list[str], event: GroupEvent) -> list[str]:
             return []
 
         return [
-            getattr(e, path[1])
+            str(getattr(e, path[1], None))
```

Comment:
Why is this one stringified?

Reply:
mypy gets angry

Reply:
Weird, could we run into a problem later on with a string of `'None'` though?

Reply:
replaced with hasattr